### PR TITLE
Create example of Add/Delete Button

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #1329
[Problem] No Add/Delete Button example
[Solution] Create new example

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

File: examples/mobile/UIComponents/components/controllers/index.html [hunk 1]
```diff
@@ -62,6 +62,11 @@ <h1>
 						On/Off switch
 					</a>
 				</li>
+				<li class="ui-li-anchor">
+					<a href="adddeletebutton.html">
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/controllers/index.html]: Could you change the name of file as `add-delete-button.html` ?

Comment (korneliakobiela) [reply examples/mobile/UIComponents/components/controllers/index.html]: Done!

File: examples/mobile/UIComponents/components/controllers/add-delete-button.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+			</h1>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-li-divider">
+					<div class="ui-li-text">
+						Add
+					</div>
+					<div class="ui-li-action">
+						<a href="#" class="ui-btn ui-btn-icon ui-icon-add" data-style="flat" ></a>
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/controllers/add-delete-button.html]: instead of `ui-icon-add` use `data-icon="add"`

Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/controllers/add-delete-button.html]: Please check `data-icon="plus"` `data-icon="minus"`

Comment (korneliakobiela) [reply examples/mobile/UIComponents/components/controllers/add-delete-button.html]: Ok, done

**Review by HunseopJeong — APPROVED** [review]
LGTM

File: examples/mobile/UIComponents/components/controllers/add-delete-button.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+			</h1>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-li-divider">
+					<div class="ui-li-text">
+						Add
+					</div>
+					<div class="ui-li-action">
+						<a href="#" class="ui-btn ui-btn-icon" data-icon="plus" data-style="flat" ></a>
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/controllers/add-delete-button.html]: class "ui-btn-icon" is not necessary

File: src/css/profile/mobile/base.less [hunk 4]
```diff
@@ -103,6 +103,8 @@
     --button-text-font-size: @button-text-font-size;
     --button-contained-text-font-size: @button-contained-text-font-size;
     --button-contained-list-text-font-size: @button-contained-list-text-font-size;
+    --btn-add-color: #00b149ff;
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/base.less]: please use #00b149 notation without alpha

File: src/css/profile/mobile/common/button.less [hunk 5]
```diff
@@ -423,21 +435,21 @@ a.ui-btn {
...
+			min-height: 32 * @px_base;
+			max-width: 32 * @px_base;
 
 			&::after {
-				width: 56 * @px_base;
-				height: 56 * @px_base;
+				width: 32 * @px_base;
+				height: 32 * @px_base;
 				border-radius: 28 * @px_base;
 				mask-size: 43 * @px_base;
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/button.less]: `mask-size` should be 32 * `@px_base`

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**